### ISSUE TITLE
Missing destination connector icons for Cumul.io, Timeplus

### ISSUE THREAD

**Issue by nataliekwong** [airbytehq/airbyte#32280]
We're missing a few connector icons for destinations in Cloud:

<img width="374" alt="Screenshot 2023-11-07 at 2 27 16 PM" src="https://github.com/airbytehq/airbyte/assets/38087517/33747d6e-5cd3-4545-8a6c-4ea5667c118e">
<img width="369" alt="Screenshot 2023-11-07 at 2 27 12 PM" src="https://github.com/airbytehq/airbyte/assets/38087517/b554a3ed-477d-4ee6-858f-50b4866906e9">

**Comment by evantahler** [comment]
Checking out new metadata viewer in the docs, both of these icons exist and are properly referenced by metadata (although the timeplus icon is janky). 
<img width="972" alt="Screenshot 2023-11-13 at 10 45 24 AM" src="https://github.com/airbytehq/airbyte/assets/303226/7020d844-29c0-409c-acb8-f9b2b0040299">
<img width="1149" alt="Screenshot 2023-11-13 at 10 46 01 AM" src="https://github.com/airbytehq/airbyte/assets/303226/c186c19a-5743-420a-9f44-47f4d979ebc3">

I think this might be a frontend or platform error that they aren't showing up.  I'll re-assign this ticket to the frontend team

**Comment by tealjulia** [comment]
Refining note:
- docs pull from metadata catalog, webapp pulls from platform code bundle (which can end up out of date)
- @bnchrch is working on a solution that will allow the webapp to consume icons directly from the metadata service

**Comment by nataliekwong** [comment]
I'm still seeing on my end those destinations are in Cloud. I defer to @bnchrch on whether this should be closed.

![Airbyte-New-destination](https://github.com/airbytehq/airbyte/assets/38087517/3982d18d-d90a-4b75-894b-57e28eff09db)

**Comment by bnchrch** [comment]
Hey @nataliekwong 

Teal is right that Ive got a PR up for making this issue redundant by loading our icons directly from the connector folder.

After a discussion with Tim, Lake, and Alex it just needs to be updated to fit the frontend teams needs.

Im coming of Oncall tomorrow and hoping to implement those changes.

In the mean time though I just created a PR to add these icons to the platform

https://github.com/airbytehq/airbyte-platform-internal/pull/10200

Which if merged will work as a stop gap until the new system is in place

**Comment by bnchrch** [comment]
TLDR: Leave this open at least until one of the two PRs is merged

**Comment by bnchrch** [comment]
@nataliekwong This is resolved in cloud (and oss for latest installs)
![image](https://github.com/airbytehq/airbyte/assets/1325608/9925b668-1c8d-40ef-81d6-5caa458cf86a)

Closing, feel free to reopen if something was missed!
